### PR TITLE
Reorganize README.md - move beta, nightly, network installer to bottom)

### PR DESCRIPTION
**Pull request by gabrielle-ong**
Preview changes: 
https://github.com/janhq/cortex.cpp/tree/chore/readme

Usage section to be added in separate PR

### CODE REVIEW

File: README.md
Comment (namchuai) [inline README.md:1]: port 3928 should be changed to 39281 I think

**Comment by namchuai** [comment]
Apart from port, looks good to me ✅ . Approved for not blocking.

**Review by namchuai — APPROVED** [review]

File: README.md
Comment (vansangpfiev) [reply README.md:1]: updated

**Comment by vansangpfiev** [comment]
Added installer links for stable.

**Pull request merged**